### ISSUE TITLE
Blazor WASM

### ISSUE THREAD

**Issue by mirmajeed1** [featbit/featbit#564]
Does this integrate with Blazor WASM project?

**Comment by cosmic-flood** [comment]
Hi @mirmajeed1 

Here's a sample that might give you some insight:
--------------------------
I've created an example in this Asp.Net Core Blazor (.NET 8) project:

![image](https://github.com/featbit/featbit/assets/68597908/869cfea5-5f35-426b-922e-ba95b3e5a2e2)

As illustrated above, I implemented a FeatureFlag component to manage the code segments I wish to control. This approach replaces the tag helper used before .NET 8.

![image](https://github.com/featbit/featbit/assets/68597908/476a301a-7664-48e8-ab63-578f158fde01)

The screenshot above shows the code for the FeatureFlag component. Within this component, it makes an API call to the backend project, which then returns the evaluated value of the feature flag to the WebAssembly client page.

P.S.: Don't forget to add `builder.Services.AddScoped` for the `HttpClient` in both the frontend and backend's `program.cs`.
--------------------------

Although I'm not an expert in Blazor development, **I am eager to receive some best practice suggestions that could help us improve quickly. Any guidance would be greatly appreciated, especially tips that can assist Blazor developers in effectively troubleshooting their problems.** 🙏🙏🙏

Thank you.
